java.lang.UnsatisfiedLinkError: /tmp/swc-jni/linux-x64-musl/libswc_jni.so: /lib/x86_64-linux-gnu/libc.so: invalid ELF header

```
[ERROR] Failed to load library integration class 'dev.yidafu.jupyper.KotlinKernelJsSupport'
/tmp/swc-jni/linux-x64-musl/libswc_jni.so: /lib/x86_64-linux-gnu/libc.so: invalid ELF header
java.lang.UnsatisfiedLinkError: /tmp/swc-jni/linux-x64-musl/libswc_jni.so: /lib/x86_64-linux-gnu/libc.so: invalid ELF header
	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:388)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:232)
	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:174)
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2389)
	at java.base/java.lang.Runtime.load0(Runtime.java:755)
	at java.base/java.lang.System.load(System.java:1953)
	at dev.yidafu.swc.SwcNative.<init>(SwcNative.kt:14)
	at dev.yidafu.jupyper.JavaScriptMagicCodeProcessor.<init>(JavaScriptMagicCodeProcessor.kt:20)
	at dev.yidafu.jupyper.KotlinKernelJsSupport.onLoaded(KotlinKernelJsSupport.kt:27)
	at org.jetbrains.kotlinx.jupyter.api.libraries.JupyterIntegration.getDefinitions(JupyterIntegration.kt:325)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner$instantiateLibraries$2$1.invoke(LibrariesScanner.kt:127)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner$instantiateLibraries$2$1.invoke(LibrariesScanner.kt:125)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner.instantiateLibraries$withErrorsHandling(LibrariesScanner.kt:110)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner.instantiateLibraries(LibrariesScanner.kt:125)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner.addLibrariesFromClassLoader(LibrariesScanner.kt:59)
	at org.jetbrains.kotlinx.jupyter.libraries.LibrariesScanner.addLibrariesFromClassLoader$default(LibrariesScanner.kt:50)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$execute$1$3.invoke(CellExecutorImpl.kt:103)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$execute$1$3.invoke(CellExecutorImpl.kt:102)
	at org.jetbrains.kotlinx.jupyter.config.LoggingKt.catchAll(logging.kt:42)
	at org.jetbrains.kotlinx.jupyter.config.LoggingKt.catchAll$default(logging.kt:41)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl.execute(CellExecutorImpl.kt:102)
	at org.jetbrains.kotlinx.jupyter.repl.CellExecutor$DefaultImpls.execute$default(CellExecutor.kt:15)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.execute(CellExecutorImpl.kt:220)
	at org.jetbrains.kotlinx.jupyter.api.libraries.CodeExecutionCallback.invoke(CodeExecutionCallback.kt:8)
	at org.jetbrains.kotlinx.jupyter.api.libraries.CodeExecutionCallback.invoke(CodeExecutionCallback.kt:6)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.execute(CellExecutorImpl.kt:242)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.runChild(CellExecutorImpl.kt:140)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.runChild(CellExecutorImpl.kt:136)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.access$runChild(CellExecutorImpl.kt:126)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext$doAddLibraries$1.invoke(CellExecutorImpl.kt:148)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext$doAddLibraries$1.invoke(CellExecutorImpl.kt:147)
	at org.jetbrains.kotlinx.jupyter.exceptions.ReplLibraryExceptionKt.rethrowAsLibraryException(ReplLibraryException.kt:30)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.doAddLibraries(CellExecutorImpl.kt:147)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.addLibraries(CellExecutorImpl.kt:182)
	at org.jetbrains.kotlinx.jupyter.api.KotlinKernelHost$DefaultImpls.addLibrary(KotlinKernelHost.kt:42)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.addLibrary(CellExecutorImpl.kt:126)
	at jupyter.kotlin.CellExtensionsKt$USE$1.invoke(cellExtensions.kt:18)
	at jupyter.kotlin.CellExtensionsKt$USE$1.invoke(cellExtensions.kt:18)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.execute(CellExecutorImpl.kt:242)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.runChild(CellExecutorImpl.kt:140)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl$ExecutionContext.processExecutionQueue(CellExecutorImpl.kt:237)
	at org.jetbrains.kotlinx.jupyter.repl.impl.CellExecutorImpl.execute(CellExecutorImpl.kt:116)
	at org.jetbrains.kotlinx.jupyter.repl.CellExecutor$DefaultImpls.execute$default(CellExecutor.kt:15)
	at org.jetbrains.kotlinx.jupyter.ReplForJupyterImpl$evalEx$1.invoke(repl.kt:479)
	at org.jetbrains.kotlinx.jupyter.ReplForJupyterImpl$evalEx$1.invoke(repl.kt:470)
	at org.jetbrains.kotlinx.jupyter.ReplForJupyterImpl.withEvalContext(repl.kt:433)
	at org.jetbrains.kotlinx.jupyter.ReplForJupyterImpl.evalEx(repl.kt:470)
	at org.jetbrains.kotlinx.jupyter.messaging.ProtocolKt$shellMessagesHandler$2$res$1.invoke(protocol.kt:318)
	at org.jetbrains.kotlinx.jupyter.messaging.ProtocolKt$shellMessagesHandler$2$res$1.invoke(protocol.kt:312)
	at org.jetbrains.kotlinx.jupyter.JupyterExecutorImpl$runExecution$execThread$1.invoke(execution.kt:37)
	at org.jetbrains.kotlinx.jupyter.JupyterExecutorImpl$runExecution$execThread$1.invoke(execution.kt:32)
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)
```

--- a/swc-binding/build.gradle.kts
+++ b/swc-binding/build.gradle.kts
@@ -8,7 +8,7 @@ plugins {
 }
 
 group = "dev.yidafu.swc"
-version = "0.4.1"
+version = "0.4.2"
 
 dependencies {
     testImplementation(kotlin("test"))

--- a/swc-binding/build.sh
+++ b/swc-binding/build.sh
@@ -1,12 +1,13 @@
 #!/bin/bash
+rm -fr ./src/main/resources/*
 
 mkdir -p ./src/main/resources/darwin-x64
 mkdir -p ./src/main/resources/darwin-arm64
-mkdir -p ./src/main/resources/darwin-universal
-mkdir -p ./src/main/resources/linux-x64-musl
+# mkdir -p ./src/main/resources/darwin-universal
+# mkdir -p ./src/main/resources/linux-x64-musl
 mkdir -p ./src/main/resources/linux-x64-gnu
-mkdir -p ./src/main/resources/win32-x64-msvc
 mkdir -p ./src/main/resources/linux-arm-gnueabihf
+mkdir -p ./src/main/resources/win32-x64-msvc
 
 # https://gist.github.com/surpher/bbf88e191e9d1f01ab2e2bbb85f9b528
 cargo build --release --target x86_64-apple-darwin
@@ -14,14 +15,14 @@ cp ./target/x86_64-apple-darwin/release/libswc_jni.dylib  ./src/main/resources/d
 cargo build --release --target aarch64-apple-darwin
 cp ./target/aarch64-apple-darwin/release/libswc_jni.dylib  ./src/main/resources/darwin-arm64
 cargo zigbuild --release --target universal2-apple-darwin
-cp ./target/universal2-apple-darwin/release/libswc_jni.dylib  ./src/main/resources/darwin-universal
+# cp ./target/universal2-apple-darwin/release/libswc_jni.dylib  ./src/main/resources/darwin-universal
 
 # Alternative zigbuild
 TARGET_CC=x86_64-linux-musl-gcc cargo build --release --target x86_64-unknown-linux-musl
-cp target/x86_64-unknown-linux-musl/release/libswc_jni.so ./src/main/resources/linux-x64-musl
+# cp target/x86_64-unknown-linux-musl/release/libswc_jni.so ./src/main/resources/linux-x64-musl
 
 cargo zigbuild --release --target arm-unknown-linux-gnueabihf
-cp target/arm-unknown-linux-gnueabihf/release/swc_jni.so ./src/main/resources/linux-arm-gnueabihf
+cp target/arm-unknown-linux-gnueabihf/release/libswc_jni.so ./src/main/resources/linux-arm-gnueabihf
 
 cargo build --release --target x86_64-unknown-linux-gnu
 cp target/x86_64-unknown-linux-gnu/release/libswc_jni.so ./src/main/resources/linux-x64-gnu

--- a/swc-binding/src/main/kotlin/dev/yidafu/swc/DllLoader.kt
+++ b/swc-binding/src/main/kotlin/dev/yidafu/swc/DllLoader.kt
@@ -81,7 +81,7 @@ object DllLoader {
 
     fun copyDll2Temp(libName: String): String {
         val jarPath = when (val p = Platform.current) {
-            is Platform.Linux -> (if (p.isArm()) "linux-arm-gnueabihf" else "linux-x64-musl") + "/lib$libName.so"
+            is Platform.Linux -> (if (p.isArm()) "linux-arm-gnueabihf" else "linux-x64-gnu") + "/lib$libName.so"
             is Platform.Mac -> (if (p.isIntel()) "darwin-x64" else "darwin-arm64") + "/lib$libName.dylib"
             is Platform.Windows -> "win32-x64-msvc/$libName.dll"
 //            Platform.SOLARIS -> TODO()

